File: Java/program/Clothing.java
//Kelas clothing anak dari product
class Clothing extends Product{
    //private atribut
    private String size;
    private String material;
    private String gender;

    //constructor
    public Clothing() {
    }
    public Clothing(String size, String material, String gender) {
        this.size = size;
        this.material = material;
        this.gender = gender;
    }
    public Clothing(String idProduct, String name, String brand, int price, String size, String material, String gender) {
        super(idProduct, name, brand, price);
        this.size = size;
        this.material = material;
        this.gender = gender;
    }

    //getter setter
    public String getSize() {
        return size;
    }
    public void setSize(String size) {
        this.size = size;
    }
    public String getMaterial() {
        return material;
    }
    public void setMaterial(String material) {
        this.material = material;
    }
    public String getGender() {
        return gender;
    }
    public void setGender(String gender) {
        this.gender = gender;
    }
}

File: Python/program/Clothing.py
#kelas Clothing anak dari kelas product

from Product import Product 

class Clothing(Product):
    __size = ""
    __material = ""
    __gender = ""

    #constructor
    def __init__(self, idProduct, name, brand, price, size, material, gender):
        super().__init__(idProduct, name, brand, price)
        self.__size = size
        self.__material = material
        self.__gender = gender

    # Getter dan Setter untuk size
    def get_size(self):
        return self.__size

    def set_size(self, size):
        self.__size = size

    # Getter dan Setter untuk material
    def get_material(self):
        return self.__material

    def set_material(self, material):
        self.__material = material

    # Getter dan Setter untuk gender
    def get_gender(self):
        return self.__gender

    def set_gender(self, gender):
        self.__gender = gender